lib/src/base/types/Configurable.vue

```diff
@@ -52,8 +52,24 @@
               <span v-if="instanceNameError" class="pagesReference mb-2">{{ $t('instanceNameRequired') }}</span>
             </div>
             <div v-if="globalSectionMode === true && pages.length > 0">
-              <div class="pagesReferenceWrapper">
-                <div class="pagesReference">{{ $t('referencedSection', {pages: pages.join(', ')}) }}</div>
+              <div class="global-sections-reference-section">
+                <div class="global-sections-first-row">
+                  <span>{{ $t('referencedSection1') }}</span>
+                  <div class="relative">
+              <span class="global-sections-pages-link" @click="showPagesList = !showPagesList">
+                {{ pages.length }}
+              </span>
+                    <div v-if="showPagesList" class="global-sections-popup">
+                      <ul>
+                        <li v-for="(page, index) in pages" :key="`refenced-pages-${index}`">{{ page }}</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <span>{{ $t('referencedSection1Pages') }}</span>
+                </div>
+                <div class="global-sections-second-row">
+                  <span>{{ $t('referencedSection2') }}</span>
+                </div>
```

Comment:
the same code repeated on multiple components.. this begs to be defined as a separate component that is used in here (and on the other components)